saleor/core/prices.py

```diff
@@ -0,0 +1,15 @@
+from decimal import Decimal
+from typing import TYPE_CHECKING, Union
+
+from babel.numbers import get_currency_precision
+
+if TYPE_CHECKING:
+    from prices import TaxedMoney, Money, TaxedMoneyRange
+
+
+def quantize_price(
+    price: Union["TaxedMoney", "Money", Decimal, "TaxedMoneyRange"], currency: str
+) -> Union["TaxedMoney", "Money", Decimal, "TaxedMoneyRange"]:
+    precision = get_currency_precision(currency)
```

Comment:
What if I run Saleor in some specific currency for example: Dogecoin?
Maby we should have some default precision?

Reply:
It is handled by `get_currency_precision`